src/components/Tabs/index.tsx

```diff
@@ -5,14 +5,11 @@ import {
 	Tabs as GrommetTabs,
 	TabsProps,
 } from 'grommet';
-export { TabProps, TabsProps } from 'grommet';
 import React from 'react';
 import asRendition from '../../asRendition';
 
 interface InnerTabProps extends TabProps {
 	compact?: boolean;
```

Comment:
I guess this prop is internal since it doesn't do anything if you set it per tab